synapse/api/constants.py

```diff
@@ -51,6 +51,7 @@ class PresenceState:
     OFFLINE = "offline"
     UNAVAILABLE = "unavailable"
     ONLINE = "online"
+    BUSY = "org.matrix.msc3026.busy"
```

Comment:
Could there be any special tricks for making sure "busy" is interpreted by both it's MSC prefix and its eventual name? `busy`, something like a frozen list with the first item being the "canonical" one.

Reply:
It'll probably be easier to have two constants, e.g. `BUSY` and `MSC3026BUSY`, once the MSC is merged (and we'll also be able to get rid of the latter after some time, once most clients are up to date with the spec).

Reply:
I suggest doing that, and changing the `==`s in your PR with `in (MSC3026BUSY, BUSY)` or something.

Reply:
Agreed, though I believe it's better to do that as a second step when the feature gets released in a stable release of the spec because implementations are not supposed to support stable notations before that point.